src/global_distrib/mod_global_distrib.erl

```diff
@@ -47,6 +48,116 @@ start(Host, Opts0) ->
 stop(Host) ->
     mod_global_distrib_utils:stop(?MODULE, Host, fun stop/0).
 
+-spec config_spec() -> mongoose_config_spec:config_section().
+config_spec() ->
+    #section{
+        items = #{<<"global_host">> => #option{type = string,
+                                               validate = domain},
+                  <<"local_host">> => #option{type = string,
+                                              validate = domain},
+                  <<"message_ttl">> => #option{type = integer,
+                                               validate = non_negative},
+                  <<"hosts_refresh_interval">> => #option{type = integer,
+                                                          validate = non_negative},
+                  <<"connections">> => connections_spec(),
+                  <<"redis">> => redis_spec(),
+                  <<"cache">> => cache_spec(),
+                  <<"bounce">> => bounce_spec()
+        },
+        required = [<<"global_host">>, <<"local_host">>]
+    }.
+
+connections_spec() ->
+    #section{
+        items = #{<<"endpoints">> => #list{items = endpoints_spec()},
+                  <<"advertised_endpoints">> => #list{items = endpoints_spec()},
+                  <<"connections_per_endpoint">> => #option{type = integer,
+                                                            validate = non_negative},
+                  <<"endpoint_refresh_interval">> => #option{type = integer,
+                                                             validate = positive},
+                  <<"endpoint_refresh_interval_when_empty">> => #option{type = integer,
+                                                                        validate = positive},
+                  <<"disabled_gc_interval">> => #option{type = integer,
+                                                        validate = positive},
+                  <<"tls">> => tls_spec()
```

Comment:
This is required, the module would crash without it. The current docs are wrong and they need correction here.